File: frameworks/csharp-dotnet/FraiseQL.Benchmark/Models/Comment.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace FraiseQL.Benchmark.Models;

[Table("tb_comment", Schema = "benchmark")]
public class Comment
{
    [Key]
    [Column("pk_comment")]
    public int PkComment { get; set; }

    [Column("id")]
    public Guid Id { get; set; }

    [Column("content")]
    [Required]
    public string Content { get; set; } = string.Empty;

    [Column("fk_post")]
    [Required]
    public int FkPost { get; set; }

    [Column("fk_author")]
    [Required]
    public int FkAuthor { get; set; }

    [Column("created_at")]
    public DateTime CreatedAt { get; set; }

    // Navigation properties
    [ForeignKey("FkPost")]
    public virtual Post? Post { get; set; }

    [ForeignKey("FkAuthor")]
    public virtual User? Author { get; set; }
}

File: frameworks/apollo-orm/src/entities/Comment.ts
import { Entity, Column, PrimaryColumn, ManyToOne, JoinColumn, CreateDateColumn } from 'typeorm';

@Entity({ name: 'benchmark.tb_comment', synchronize: false })
export class Comment {
  @PrimaryColumn({ type: 'varchar' })
  id!: string;

  @Column({ type: 'integer' })
  pk_comment!: number;

  @Column({ type: 'integer' })
  fk_post!: number;

  @Column({ type: 'integer' })
  fk_author!: number;

  @Column({ type: 'text' })
  content!: string;

  @CreateDateColumn({ type: 'timestamp' })
  created_at!: Date;

  // Relationships - using string references to avoid circular dependencies
  @ManyToOne('Post', 'comments')
  @JoinColumn({ name: 'fk_post', referencedColumnName: 'pk_post' })
  post?: any;

  @ManyToOne('User', 'comments')
  @JoinColumn({ name: 'fk_author', referencedColumnName: 'pk_user' })
  author?: any;
}